src/core/middlewares/middlewares.go

```diff
@@ -82,9 +67,9 @@ func MiddleWares() []beego.MiddleWare {
 		csrf.Middleware(),
 		security.Middleware(),
 		readonly.Middleware(readonlySkippers...),
-		orm.Middleware(legacyAPISkipper),
+		orm.Middleware(),
```

Comment:
@heww Is this check not required now ?

Reply:
I think this skipper does not work for legacy APIs now.